src/toplevel_generators.jl

```diff
@@ -133,10 +133,14 @@ function generate_wrapper_load(src_name, pkg_uuid, __source__)
             # Helper function to parse triplets for us
             $(platform_parse_compat())
             function make_wrapper_dict(dir, x)
-                return Dict(
-                    parse_wrapper_platform(basename(f)[1:end-3]) =>
-                        joinpath(dir, "wrappers", f) for f in x
-                )
+                # (1) make the Dict type inferrable
```

Comment:
For this point, using `Dict{Platform,String}( <generator> )` would be sufficient, right?

Reply:
Yes, specifying the exact `Dict` type would fix that.